py-kornia/Cargo.toml

```
@@ -0,0 +1,15 @@
+[package]
+name = "kornia-py"
+version = "0.0.9"
```

in my eyes, this refactor is too big to only bump a patch number 
```suggestion
version = "0.1.0"
```